mwdb/web/src/components/AttributesAddModal.js

```diff
@@ -183,14 +196,35 @@ export default function AttributesAddModal({ isOpen, onAdd, onRequestClose }) {
                                 wrapEnabled
                                 onChange={(input) => setAttributeValue(input)}
                                 value={attributeValue}
-                                width="300px"
+                                width="500px"
                                 height="150px"
                                 setOptions={{
                                     useWorker: false,
                                 }}
                             />
                         )}
                     </div>
+                    {richTemplate ? (
+                        <div className="form-group">
+                            <label>Rich attribute preview</label>
+                            <table
+                                className="table table-striped table-bordered table-hover data-table"
+                                style={{
+                                    width: `500px`,
+                                }}
+                            >
+                                <tbody>
+                                    <RichAttributeRenderer
+                                        template={richTemplate}
+                                        value={attributeValue}
```

Comment:
We need to handle `string`-typed attributes correctly.
```suggestion
                                        value={attributeType === "string" ? JSON.stringify(attributeValue) : attributeValue}
```